### PR TITLE
Feature/48058 Migrate radio

### PR DESCRIPTION
**Pull request by jszczech**

### CODE REVIEW

File: components/02-elements/form/radio/Radio.stories.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  .add('Default', () => ({
+    components: { App, AlpacaRadio },
+    data() {
+      return {
+        options
+      }
+    },
+    template: `
+      <app>
+        <alpaca-radio
```
Comment (Igloczek) [inline components/02-elements/form/radio/Radio.stories.js]: There is no way to get any value from these inputs.

Comment (Igloczek) [inline components/02-elements/form/radio/Radio.stories.js]: There is no way to get any value from these inputs.

File: components/02-elements/form/radio/Radio.stories.js [hunk 2]
```diff
@@ -20,7 +21,9 @@ storiesOf('Elements/Form/Radio', module)
           :legend="options.legend" 
           :options="options.options" 
           :name="options.name"
+          v-model="selected"
         />
+        Selected option: {{ selected }}
```
Comment (Igloczek) [inline components/02-elements/form/radio/Radio.stories.js]: I don't think we need it, we can preview this value in Storybook, via https://github.com/Sambego/storybook-state or https://github.com/storybooks/storybook/tree/master/addons/knobs

File: components/02-elements/form/radio/Radio.vue [hunk 3]
```diff
@@ -35,7 +38,15 @@
 
 <script>
   export default {
+    model: {
+      prop: 'checked',
+      event: 'change'
+    },
     props: {
+      checked: {
+        type: [String, Object, Number, Boolean],
```
Comment (Igloczek) [inline components/02-elements/form/radio/Radio.vue]: Huh, why do we need that many types?

File: components/02-elements/form/radio/Radio.stories.js (hunk 2 above)
Comment (Igloczek) [reply components/02-elements/form/radio/Radio.stories.js]: But can be done as a enhancement later, so let's just create a task for this and do to next issue :)

File: components/02-elements/form/radio/Radio.vue (hunk 3 above)
Comment (jszczech) [reply components/02-elements/form/radio/Radio.vue]: This component can accept any of these types.

Comment (Igloczek) [reply components/02-elements/form/radio/Radio.vue]: @jszczech 👀

Comment (jszczech) [reply components/02-elements/form/radio/Radio.vue]: This name is misleading, maybe it should be e.g. 'checkedValue' (i changed it). Values can be different than boolean. 

I checked it in some other library, like bootstrap-vue: 

https://github.com/bootstrap-vue/bootstrap-vue/blob/2e5422aa30cd6197846e66016606d9f3e1455f84/src/components/form-radio/form-radio.js#L29

There also added more types than just boolean.

**Pull request merged**